armi/physics/neutronics/globalFlux/tests/test_globalFluxInterface.py

```diff
@@ -107,6 +107,7 @@ def test_interaction(self):
         Check that a 1000 pcm rx swing is observed due to the mock.
         """
         cs = settings.Settings()
+        cs = cs.modified(newSettings={"burnSteps": 4})
```

Comment:
This makes it seem like maybe we need better defaults.  Or am I missing something?

Reply:
Outdated. These types of changes were able to be unwound after the settings defaults were reinstated.